src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToLatexFormatterTest.java

```diff
@@ -45,7 +45,7 @@ public void testHTML() {
         assertEquals("{\\\"{a}}", formatter.format("&auml;"));
         assertEquals("{\\\"{a}}", formatter.format("&#228;"));
         assertEquals("{\\\"{a}}", formatter.format("&#xe4;"));
-        assertEquals("{$\\Epsilon$}", formatter.format("&Epsilon;"));
+        assertEquals("{{$\\Epsilon$}}", formatter.format("&Epsilon;"));
```

Comment:
And one will need to distinguish between LaTeX and BibTeX. It is like writing "the correct way to write JabRef in LaTeX is `JabRef` not `{JabRef}`!".

Finally, do not forget that lots of people still use BibTeX and not Biblatex. For lots of reasons. And if Biblatex had a good enough converter it would produce `ä` not `{ä}`. JabRef will convert `{{{{{{{\"{{{{a}}}}}}}}}}}`to `ä`. Despite using a non-XML format of the conversion table.